Sources/MountebankSwift/Models/Stub/Response/Is/Is.swift

```diff
@@ -3,8 +3,9 @@ import Foundation
 // swiftlint:disable type_name
 /// A regular predefined response. Merges the specified response fields with ``Imposter``.`defaultResponse`
 public struct Is: StubResponse, Equatable {
-    public static var defaultBehaviors: [Behavior] = []
-    public static var defaultHeaders: [String: String] = [:]
+    
+    public private(set) static var defaultBehaviors: [Behavior] = []
```

Comment:
This still does give some warnings, but only fixable to make `Is` an actor, mark it with `@MainActor`, or remove the static props.

Reply:
I think we can remove them since I think that you can just use `Imposter.defaultResponse` for these things. But we have to test that first.

Reply:
`defaultResponse` does something different than this `defaultBehaviors` property. Will keep it for now like this.